components/servo/lib.rs

```diff
@@ -228,6 +228,7 @@ where
         feature = "tracing",
         tracing::instrument(skip(embedder, window), fields(servo_profiling = true))
     )]
+    #[allow(clippy::new_ret_no_self)]
```

Comment:
```suggestion

    #[allow(clippy::new_ret_no_self)]
```

Reply:
@mrobinson do you suggest I add an empty line before the ```allow``` here?

Reply:
When I add the empty line, ```test-tidy``` reports some issues from  there.

Reply:
Apologies. My suggestion was a mistake. Please discard it.